test/test_resume.cpp

```diff
@@ -207,7 +207,8 @@ void default_tests(torrent_status const& s, lt::time_point const time_now)
 	TEST_CHECK(s.active_time >= 1339);
 	TEST_CHECK(s.active_time < 1339 + 10);
 
-	auto const now = duration_cast<seconds>(time_now.time_since_epoch()).count();
+	auto const now = time(NULL);
```

Comment:
this doesn't look right. `now` is only used to compute relative time durations, why would this make a difference?